test/metabase/query_processor/middleware/add_implicit_clauses_test.clj

```diff
@@ -152,6 +152,18 @@
                          {:source-query    source-query
                           :source-metadata source-metadata}))))))))
 
+(deftest ^:parrallel expression-with-only-field-in-source-query-test
```

Comment:
I was comparing diffs between this and the backport. This seems to be a typo.

```diff
- (deftest ^:parrallel expression-with-only-field-in-source-query-test
+ (deftest ^:parallel expression-with-only-field-in-source-query-test
```

Reply:
Yes. Found that while backporting. Will correct in master shortly.

Reply:
The linter should have complained.